Examples/Idealgas.py

```diff
@@ -0,0 +1,150 @@
+from openmmtools.testsystems import IdealGas
+from openmm import unit
+
+"""
+This example explore an ideal gas system, where the particles are non-interacting. 
+This will use the MonteCarloBarostatMove to sample the volume of the system and 
+MetropolisDisplacementMove to sample the particle positions.
+
+This utilizes the IdealGas example from openmmtools to initialize particle positions and topology.
+
+"""
+
+# Use the IdealGas example from openmmtools to initialize particle positions and topology
+# For this example, the topology provides the masses for the particles
+
+n_particles = 216
+temperature = 298 * unit.kelvin
+pressure = 1 * unit.atmosphere
+
+ideal_gas = IdealGas(nparticles=n_particles, temperature=temperature, pressure=pressure)
+
+
+from chiron.potential import IdealGasPotential
```

Comment:
`IdealGas` is an openmmtools test system, so it does not contain a compatible potential function.  

I think it would be good to reimplement the tests we will be using as "chiron" tests that would provide a more convenient interface for us and fewer dependencies.  I created an issue for this #30 that we can address in a separate PR.